docs/geedocs/answer/3481558.html

```diff
@@ -127,6 +127,9 @@
 <code><b>--editvolumes</b> [--assetroot <i>path</i>]</code></pre>
 <p><i>Optional</i>. Follow the prompts to add a volume to the selected asset root or, modify the <code>localpath</code> definition for an existing volume, or to add a volume definition.</p>
 <pre>
+<code><b>--listvolumes</b> [--assetroot <i>path</i>]</code></pre>
+<p><i>Optional</i>. List the available (configured) volumes for the selected asset root.</p>
```

Comment:
Overall I think this would be better incorporated when #311 goes in.

Right now there is nothing that suggests it would "magically" reach out to another host to list volumes or do any other operation, and command line tools usually work on the host they're run on.